README.md

```diff
@@ -25,154 +22,334 @@
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_Air_pollution"
+- Full name: Air Pollution
+- Description: Combined air pollution concentrations (PM10, PM2.5, NO2, BaP, BzN)
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_PM10_air_quality_PM10_norm_round"
+- Full name: Particulate matter 10
+- Description: Concentration of particulate matter PM10
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_PM25_air_quality_PM25_normnew"
+- Full name: Particulate matter 2.5
+- Description: Concentration of particulate matter PM2.5
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_NO2_air_quality_NO2_norm_rounded"
+- Full name: Nitrogen dioxide pollution levels
+- Description: Concentration of nitrogen dioxide NO2
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_BaP_air_quality_norm"
+- Full name: Benzo(a)pyrene pollution levels
+- Description: Concentration of benzopyrene Bap pollutant
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_BzN_final_newgrid_BZN_norm"
+- Full name: Benzene pollution levels
+- Description: Concentration of benzene Bap pollutant
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+"combined_layer_QoL_Percentage_tree_cover_perc_treecover"
+- Full name: Percentage of Tree Cover
+- Description: Percentage of Tree Cover
+- Type of variable: quantitative normalized
+- Unit of measurement: 0-1 (low to high)
+
+4. Specialised formats or other abbreviations used:
+N/A
+
+6. Total file size:
+315 KB
 
 # 5. SHARING/ACCESS INFORMATION
 ## 5.1 Licenses/restrictions placed on the data:
-...
+CC BY 4.0
 
 ## 5.2 Links to other resources:
+N/A
 
 ### 5.2.1 Links to publications that cite or use the data:
-...
+- [Targeted Urban Stream Renewal: The Case of Bílá Nisa in Jablonec nad Nisou](https://github.com/sdgis-edu-tud/report-asa2025-groupd.git)
 
 ### 5.2.2 Links to other publicly accessible locations of the data: 
-...
+- N/A
 
 ### 5.2.3 Links/relationships to ancillary data sets: 
-...
+- N/A
 
 ### 5.2.4 Links to publicly accessible scripts for analysis of the dataset:
-- [Link title](link url)
+- [Targeted Urban Stream Renewal: The Case of Bílá Nisa in Jablonec nad Nisou](https://github.com/sdgis-edu-tud/report-asa2025-groupd.git)
 
 ### 5.2.5 Was data derived from another source?
-Yes/No
+Yes
 
 ## 5.3 Recommended citation for this dataset:
-...
+van Asch, F., Auer, F., Nova, M. J., Lin, R. & Kunigėlytė, U. N. (2025). Targeted Urban Stream Renewal: The Case of Bílá Nisa in Jablonec nad Nisou. TU Delft Faculty of Architecture & The Built Environment. Retrieved from https://github.com/sdgis-edu-tud/report-asa2025-groupd.git
```

Comment:
When you start filling in the form on 4TU.ResearchData, you can reserve the DOI at the draft stage. Make sure you update the link in the citation with the DOI link before submission.